brightcove-exoplayer/PulseSampleApp/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,13 @@
+<resources>
+    <string name="app_name">PulseSampleApp</string>
+
+    <!-- A sample Brightcove Edge Account ID -->
+    <string name="account">3636334163001</string>
+
+    <!-- A sample Brightcove Edge Policy Key -->
+    <string name="policy">BCpkADawqM1W-vUOMe6RSA3pA6Vw-VWUNn5rL0lzQabvrI63-VjS93gVUugDlmBpHIxP16X8TSe5LSKM415UHeMBmxl7pqcwVY_AZ4yKFwIpZPvXE34TpXEYYcmulxJQAOvHbv2dpfq-S_cm</string>
+
+    <!-- A sample Brightcove Video ID -->
+    <string name="videoId">3637773814001</string>
+    <string name="pulse_host_url">https://pulse-demo.videoplaza.tv</string>
+</resources>
```

Comment:
We should get into the habit of using the Native SDK Demo account, instead of our test accounts.
AccountID: 5420904993001
PolicyKey: BCpkADawqM1RJu5c_I13hBUAi4c8QNWO5QN2yrd_OgDjTCVsbILeGDxbYy6xhZESTFi68MiSUHzMbQbuLV3q-gvZkJFpym1qYbEwogOqKCXK622KNLPF92tX8AY9a1cVVYCgxSPN12pPAuIM
Video ID: 5421538222001